gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -301,9 +313,14 @@ gst_tensor_filter_install_properties (GObjectClass * gobject_class)
       g_param_spec_string ("sub-plugins", "Sub-plugins",
           "Registrable sub-plugins list", "",
           G_PARAM_READABLE | G_PARAM_STATIC_STRINGS));
-  g_object_class_install_property (gobject_class, PROP_NNAPI,
-      g_param_spec_string ("nnapi", "NNAPI",
-          "Enable Neural Newtorks API ?", "", G_PARAM_READWRITE));
+  g_object_class_install_property (gobject_class, PROP_ACCELERATOR,
+      g_param_spec_string ("accelerator", "ACCELERATOR",
+          "Set accelerator for the subplugin with format "
+          "(true/false):(comma separated ACCELERATOR(s)). "
+          "true/false determines if accelerator is to be used. "
+          "list of accelerators determines the backend (ignored with false). "
+          "Example, if GPU and NPU can be used but not CPU - (GPU,NPU,!CPU).",
```

Comment:
```(true/false):(comma separated ACCELERATOR(s))```
<->
```(GPU,NPU,!CPU)```
doesn't look consistent.

Reply:
updated the description to `true:(GPU,NPU,!CPU)`